rollout.py

```diff
@@ -170,6 +184,8 @@ def _main():
             help='rollout start (e.g. "10 am")')
     add.add_argument('duration', metavar='duration-hours', type=int,
             help='rollout duration (e.g. "48")')
+    add.add_argument('--skip-build-id-check', action='store_true',
+                     help='skip validiting build IDs')
```

Comment:
`validating`

Hmm, the rest of the code calls these `versions`.  We should probably be consistent; which name should we use?